internal/cmd/build/go.sum

```diff
@@ -3,18 +3,27 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/BurntSushi/toml v1.4.1-0.20240526193622-a339e1f7089c h1:pxW6RcqyfI9/kWtOwnv/G+AzdKuy2ZrqINhenH4HyNs=
```

Comment:
All changes to `go.mod` and `go.sum` are the result of running `go get gotest.tools/gotestsum` (looks like it was already a transitive dependency IIUC).